### DISCUSSION TITLE
Upgrade to 2.0.0-beta - missing bundles (no CSS)

### DISCUSSION THREAD

**Discussion by VOOM108** [kimai/kimai#3756] [On-Premise: Installation & Updates] [answered]
I upgraded from 1.19 stable to 2.0.0-beta and everything seems to have gone well - I can log in, no 500 errors, data is there etc. 

What is missing is CSS - in the console it says: 

https://zeit2.108design.com/bundles/tabler/tabler.css net::ERR_ABORTED 404

Is there a way to fix this? 

(Ubuntu / Plesk / PHP 8.1)

**Comment by kevinpapst** [comment, marked as answer]
That is weird, composer usually executes `bin/console assets:install` after `composer update`.
Did you have a hickup there? Try to run it manually.

**Reply by VOOM108** [reply]
Thanks, worked :D

**Reply by kevinpapst** [reply]
That's good. Even better if we find out what cause the step to break... did anything unexpected happen during the update?

**Reply by VOOM108** [reply]
It was "messy" as it always is with me, since I hardly touch the cli over long periods of time. A miracle, it worked out in the end. I actually used the wrong database at first, and also forgot to copy the level above /kimai in the file system from the previous version. Only when I could not log in, as I used to have different user credentials, I realized, it was the old database. So at that point, otherwise already upgraded, I set the right database into .env and went on from there. 

How and at what point the composer update was incomplete I could not even guess. So I would presume: The issue was once more sitting in front of the screen....

**Reply by kevinpapst** [reply]
Hahaha, okay. Thanks for sharing your insights. 
I assume it feels scary to update, sorry that the process is not more user-friendly.

**Reply by VOOM108** [reply]
Actually, that was probably the most uncomplicated update/upgrade so far. It took less time, and in the end everything I set up in 1.x is still working, including invoice-templates etc. This is the first bigger update where I did not have to start over with a fresh installation without data. And though the process was bumpy, the only thing left not working before I had to ask here, was what you solved easily in this thread. :)
